packages/loading-skeleton/package.json

```diff
@@ -33,6 +33,7 @@
   "license": "MIT",
   "dependencies": {
     "@kaizen/component-base": "^1.1.0",
+    "@kaizen/typography": "2.1.0",
```

Comment:
Hm, `typography` is just used for the stories, isn't it? Could we make it a dev dependency then? Would that work for storybook?

Reply:
I am using it for the `HeadingVariants`  within LoadingHeading.tsx as well.